src/Api.Rest/Api.Rest.csproj

```diff
@@ -58,7 +58,8 @@
     <PackageReference Include="Microsoft.IdentityModel.Logging" Version="6.11.1" />
     <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="6.11.1" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="Newtonsoft.Json.Bson" Version="1.0.2" />
+    <PackageReference Include="Newtonsoft.Json.Bson" Version="1.0.2" />
+    <PackageReference Include="System.Text.Json" Version="6.0.5" />
```

Comment:
Important: you need to add new dependencies to the build.cake file, e.g. line 189 for Api.Rest.Dtos and 238 for Api.Rest. This is currently needed to add the dependencies to the Appveyor build.

Reply:
I tried to test the new nuget (with correct build.cake), but there seems to be a problem on write requests (POST/PUT) with System.Text.Json. The request seems to hang when copying the stream to the server. Retrieving data however works fine. I dont know exactly why, since serializing seems to work correctly. Maybe you can find out more.

Reply:
Thank you for the hint. I added the dependencies in build.cake.

I will try out and hope I can reproduce the issue, I will come on you.

Reply:
@czthiele ab47d59194c74065304cf3c8eabd82e9c671832a fixes the freeze on write requests.